rasa/core/policies/ted_policy.py

```diff
@@ -319,36 +327,35 @@ def _standard_featurizer(max_history: Optional[int] = None) -> TrackerFeaturizer
 
     def __init__(
         self,
-        featurizer: Optional[TrackerFeaturizer] = None,
-        priority: int = DEFAULT_POLICY_PRIORITY,
-        max_history: Optional[int] = None,
+        config: Dict[Text, Any],
+        model_storage: ModelStorage,
+        resource: Resource,
+        execution_context: ExecutionContext,
         model: Optional[RasaModel] = None,
-        fake_features: Optional[Dict[Text, List["Features"]]] = None,
+        featurizer: Optional[TrackerFeaturizer] = None,
+        fake_features: Optional[Dict[Text, List[Features]]] = None,
         entity_tag_specs: Optional[List[EntityTagSpec]] = None,
-        should_finetune: bool = False,
-        **kwargs: Any,
     ) -> None:
         """Declares instance variables with default values."""
         self.split_entities_config = rasa.utils.train_utils.init_split_entities(
-            kwargs.get(SPLIT_ENTITIES_BY_COMMA, SPLIT_ENTITIES_BY_COMMA_DEFAULT_VALUE),
-            self.defaults.get(
+            config.get(SPLIT_ENTITIES_BY_COMMA, SPLIT_ENTITIES_BY_COMMA_DEFAULT_VALUE),
+            self.default_config.get(
                 SPLIT_ENTITIES_BY_COMMA, SPLIT_ENTITIES_BY_COMMA_DEFAULT_VALUE
             ),
         )
 
-        # TODO: check if the else statement can be removed.
+        super().__init__(
+            config, model_storage, resource, execution_context, featurizer=featurizer
+        )
+
+        # TODO: check if this statement can be removed.
         #  More context here -
         #  https://github.com/RasaHQ/rasa/issues/5786#issuecomment-840762751
-        if not featurizer:
```

Comment:
This is now covered by `_featurizer_from_policy_config` in `super().__init__`?

Reply:
yep because I move the `super()` part before this part.